lib/rubocop/cop/rails/migration_class_name.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop makes sure that each migration file defines a migration class
+      # whose name matches the file name.
+      # (e.g. `20220224111111_create_users.rb` should define `CreateUsers` class.)
+      #
+      # @example
+      #   # db/migrate/20220224111111_create_users.rb
+      #
+      #   # bad
+      #   class SellBooks < ActiveRecord::Migration[7.0]
+      #   end
+      #
+      #   # good
+      #   class CreateUsers < ActiveRecord::Migration[7.0]
+      #   end
+      #
+      class MigrationClassName < Base
+        extend AutoCorrector
+
+        MSG = 'Replace with `%<preferred_class_name>s` that matches the file name.'
```

Comment:
Maybe "corrected" is better than "preferred" . Because it ’s not a difference in style :-)

```suggestion
        MSG = 'Replace with `%<corrected_class_name>s` that matches the file name.'
```